src/sections/Community/Handbook/about.js

```diff
@@ -33,6 +34,24 @@ const Intro= () => {
             <h2>Newcomer’s Path</h2>
             <p>Thank you for your interest in contributing to Layer5 open source initiatives!</p>
             <p>In combination with the Layer5 Community Guide and Contributing Guide, the following set of instructions are to encourage and support first-time (or near first-time) open-source contributors.</p>
+            <Row className="newcomers-journey">
+            <Col
+              className="heading"
+              sm={12}
+              lg={6}
+              onMouseOver={() => setHover(true)}
+              onMouseOut={() => setHover(false)}
```

Comment:
If you are copying the code, always do confirm how the code is working there and what all things should be copied.
There is a `useState` function defined for `setHover` in the code from where the code is taken, and you haven't 
 added that here.

Reply:
Also, where have you redefined the CSS being used there under each classNames ?